index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Index: ONEI tools</title>
+
+    <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="div-header">
+            ONEI│tools
+            <div class="info-perfil">
+                <p> <b>Perfil:</b> Administrator</p>
+                <p>ROLE_ADMIN</p>
+                <p>_ Salir</p>
+            </div>
+        </div>
+        <nav class="nav-header">Inicio /</nav>
+    </header>
+
+        <aside class="aside">
+            <input type="search" name="" id="aside-input" class="search-aside" placeholder="_ Buscar">
+
+            <a id="aside" href="corte_tabla.html">_ Tabla</a>
+            <a id="aside" href="abc">_ Comparación</a>
+            <a id="aside" href="abc">_ CheMesAn</a>
+            <a id="aside" href="abc">_ Consultas</a>
+            <a id="aside" href="abc">_ Convenio Informativo</a>
+            <a id="aside" href="abc">_ Plan de Procesamiento</a>
+            <a id="aside" href="abc">_ Formularios vacíos</a>
+            <a id="aside" href="abc">_ Graficar</a>
+            <a id="aside" href="abc">_ Cambiar conexión</a>
+            <a id="aside" href="abc">_ Gestionar Captación</a>
+            <a id="aside" href="abc">_ Configuración</a>
+            <a id="aside" href="abc">_ Usuarios</a>
+            <a id="aside" href="abc">_ Ayuda!</a>
+        </aside>
+    
+    <div class="div-content-page">
+        <input type="search" name="" id="" class="search-main" placeholder="_ ¿Qué le gustaría buscar?">
```

Comment:
si no tienen valor no se ponen
```suggestion
        <input type="search" class="search-main" placeholder="_ ¿Qué le gustaría buscar?" />
```

Reply:
jajaja, es que yo se los dejo así para "recordarme" que salen así por defecto; manías que le salen a uno programando.
Ya lo sé y lo trataré de recordar pa la próxima!!